pyclaragenomics/setup.py

```diff
@@ -83,17 +33,14 @@ def get_installed_path(self, component=""):
             os.path.join(cmake_root_dir, "cudapoa/include"),
```

Comment:
these can become `cga_install_dir/include/claragenomics/cudaaligner` and `cga_install_dir/include/claragenomics/cudapoa`